metatensor-core/tests/cpp/data.cpp

```diff
@@ -1,106 +1,352 @@
-#include <memory>
+#include <vector>
 
 #include <catch.hpp>
 
+// This must be included for the DLManagedTensor definition
+#include <dlpack/dlpack.h>
```

Comment:
Isn't this included by `metatensor.hpp`?

Reply:
Transiently via `metatensor.h`